cdc/deployments/tikv-cdc/docker/integration-test.Dockerfile

```diff
@@ -29,32 +29,30 @@ RUN yum install -y \
 	git \
 	bash-completion \
 	wget \
-    which \
+	which \
 	gcc \
 	make \
-    curl \
-    tar \
-    musl-dev \
-    psmisc
+	curl \
+	tar \
+	musl-dev \
+	psmisc \
+	mysql
 RUN wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
 RUN yum install -y epel-release-latest-7.noarch.rpm
 RUN yum --enablerepo=epel install -y s3cmd
-RUN wget -i -c http://dev.mysql.com/get/mysql57-community-release-el7-10.noarch.rpm
-RUN yum install -y mysql57-community-release-el7-10.noarch.rpm
-RUN yum install -y mysql-server
 
 # Copy go form downloader.
 COPY --from=downloader /usr/local/go /usr/local/go
 ENV GOPATH /go
 ENV GOROOT /usr/local/go
 ENV PATH $GOPATH/bin:$GOROOT/bin:$PATH
 
-WORKDIR /go/src/github.com/pingcap/tiflow
+WORKDIR /go/src/github.com/tikv/migration/cdc
 COPY . .
 
-# Clean bin dir and build TiCDC.
+# Clean bin dir and build TiKV-CDC.
 # We always need to clean before we build, please don't adjust its order.
 RUN make clean
-RUN make integration_test_build kafka_consumer cdc
-COPY --from=downloader /root/download/bin/* ./bin/
+RUN make integration_test_build cdc
```

Comment:
The `tikv-cdc` is necessary as test cases using `$(tikv-cdc cli xxx)` ?

Reply:
I mean `make integration_test_build cdc` -> `make integration_test_build`, because `make integration_test_build` will build `tikv-cdc.test`  & `tikv-cdc`

Reply:
Oh, I see.
Fixed, PTAL~